app/views/submissions/_correction.html.erb

```diff
@@ -1,23 +1,39 @@
 <div class="row">
   <div class="col-12">
-    <%= link_to t('buttons.upload'),
-                edit_correction_path(submission),
-                class: 'btn btn-sm btn-outline-secondary mb-2',
-                remote: true %>
-    <span class="correction-action-area"
-         data-id="<%= submission.id %>">
-      <% if submission.correction %>
-        <%= link_to correction_display_mode(submission),
-                    show_correction_path(submission),
-                    class: 'btn btn-sm btn-outline-secondary mb-2',
-                    data: { turbolinks: false } %>
-        <%= link_to t('buttons.delete'),
-                    delete_correction_path(submission),
-                    class: 'btn btn-sm btn-outline-secondary mb-2',
-                    data: { confirm: t('confirmation.generic') },
-                    method: :delete,
-                    remote: true %>
-      <% end %>
-    </span>
+    <% if !current_user.editor_or_teacher_in?(@lecture) %>
```

Comment:
I think that this is a bug: It is possible that someone is an editor of the lecture and a tutor at the same time (e.g. if the "Obertutor" also has a tutorial which happens sometimes). In this logic this person would not have access to the "tutor's buttons".

Reply:
should we grant the lecturer/"Obertutor" all access to the submission/correction process?